src/wp-includes/compat.php

```diff
@@ -539,6 +539,46 @@ function array_all( array $array, callable $callback ): bool { // phpcs:ignore U
 	}
 }
 
+if ( ! function_exists( 'array_first' ) ) {
+	/**
+	 * Polyfill for `array_first()` function added in PHP 8.5.
+	 *
+	 * Returns the first element of an array.
+	 *
+	 * @since 6.8.3
+	 *
+	 * @param array $array The array to get the first element from.
+	 * @return mixed|null The first element of the array, or null if the array is empty.
+	 */
+	function array_first( array $array ) { // phpcs:ignore Universal.NamingConventions.NoReservedKeywordParameterNames.arrayFound
+		if ( empty( $array ) ) {
+			return null;
+		}
+
+		return $array[ array_key_first( $array ) ];
```

Comment:
I think this can be done without the need for the additional function call.

```suggestion
		foreach ( $array as $value ) {
			return $value;
		}
```

Reply:
got it updating to use this